app/util/mapIconUtils.js

```diff
@@ -667,10 +663,6 @@ export function drawCitybikeIcon(
     const smallCircleRadius = 11 * tile.scaleratio;
     x = geom.x / tile.ratio - width + smallCircleRadius * 2;
     y = geom.y / tile.ratio - height;
-    const showAvailabilityBadge =
-      showAvailability &&
-      (bikesAvailable || bikesAvailable === 0) &&
-      state === BIKESTATION_ON;
     let icon = `${iconName}_station_${color}_large`;
```

Comment:
if the reason for not showing badges is the missing state, we should include it in this check, no?

Reply:
I'm adding it back with an additional condition that allows for null states. I guess the real solution should be in the data and not in the condition.